package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "react-native-onesignal",
-  "version": "4.2.0",
+  "version": "4.2.1",
```

Comment:
New API, `OneSignal.setLanguage` was added. We should bump minor version value to be 4.3.0 for this one.